### PR TITLE
ETD-272: Add existing scripts -Generate MARCXML

### PR DESCRIPTION
**Pull request by cgoines**
**Add existing scripts -Generate MARCXML**
* * *

**JIRA Ticket**: https://jira.huit.harvard.edu/browse/ETD-272

# What does this Pull Request do?
this copies etd mets into marcxml for alma

# How should this be tested?

0) copy .env.example to .env, set it to dev values
1) spin up instance locally
2) enter container
3) `pytest` - all tests should pass

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? y
- integration tests? n

# Interested parties
Tag (@ mention) interested parties: @awoods

### CODE REVIEW

File: tests/unit/test_worker.py [hunk 1]
```diff
@@ -37,3 +62,83 @@ def mock_get(*args, **kwargs):
...
+    def test_writeMarcXml(self, monkeypatch):
+        batch = "alma2023071720-993578-gsd"
+        batchOutputDir = "./tests/data/in/proquest2023071720-993578-gsd"
+        verbose = False
+        global generatedMarcXmlValues
+        writeMarcXml(batch, batchOutputDir, generatedMarcXmlValues, verbose)
+        metsFile = batchOutputDir + "/" + batch + ".xml"
+        assert os.path.exists(metsFile)
+        namespace_mapping = {'marc': 'http://www.loc.gov/MARC21/slim'}
+        doc = ET.parse(metsFile)
```
Comment (awoods) [inline tests/unit/test_worker.py:106]: Should this be `marcFile` instead of `metsFile`?

**Review by awoods — CHANGES_REQUESTED** [review]
I would like to also look into the flake8 issues before we merge.

File: tests/unit/test_worker.py (hunk 1 above)
Comment (cgoines) [reply tests/unit/test_worker.py]: updated

Comment (cgoines) [reply tests/unit/test_worker.py]: i did fix the flake8 issues in the test_worker.py

**Review by awoods — APPROVED** [review]

**Pull request merged**